ixmp/reporting/utils.py

```diff
@@ -211,16 +211,19 @@ class AttrSeries(pd.Series):
     _metadata = ['attrs']
 
     def __init__(self, *args, **kwargs):
-        if isinstance(args[0], xr.DataArray):
+        if 'attrs' in kwargs:
+            # Use provided attrs
+            attrs = kwargs.pop('attrs')
+        elif hasattr(args[0], 'attrs'):
```

Comment:
not sure if this is the exact right way to do things, but I had a test fail because `attrs` was being passed in the constructor with `args[0]` was an `xr.DataArray`.